Sources/RemoteMessaging/Model/RemoteConfigModel.swift

```diff
@@ -20,5 +20,15 @@ import Foundation
 
 public struct RemoteConfigModel {
     let messages: [RemoteMessageModel]
-    let rules: [Int: [MatchingAttribute]]
+    let rules: [RemoteConfigRule]
+}
+
+public struct RemoteConfigRule {
+    let id: Int
+    let targetPercentile: RemoteConfigTargetPercentile?
+    let attributes: [MatchingAttribute]
+}
+
+public struct RemoteConfigTargetPercentile {
+    let before: Float?
```

Comment:
This is the percentile that will be sampled _before_ we apply the attributes. The alternative here is that we do the sampling after matching attributes, but it was agreed that we'll only support the `before` case for now.